### PR TITLE
replay: minReplayDuration

### PR DESCRIPTION
**Pull request by bruno-garcia**
Feature added by @mydea through: https://github.com/getsentry/sentry-javascript/pull/8596

cc @billyvg

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

File: src/platforms/javascript/common/session-replay/configuration.mdx [hunk 1]
```diff
@@ -25,6 +25,7 @@ The following can be configured as integration options in `new Replay({})`:
 | stickySession           | boolean                  | `true`   | Keep track of users across page loads. Note, because closing a tab ends the session, a single user using multiple tabs will be recorded as multiple sessions. |
 | mutationLimit           | number                   | 10000    | The upper bound of mutations to process before Session Replay stops recording due to performance impacts. See [Mutation Limits](#mutation-limits)             |
 | mutationBreadcrumbLimit | number                   | 750      | The upper bound of mutations to process before Session Replay sends a breadcrumb to warn of large mutations. See [Mutation Limits](#mutation-limits)          |
+| minReplayDuration       | number                   | 5000     | The length of the replay, **in milliseconds**, before the SDK should start flushing to Sentry. Max value is 15000.                                            |
```
Comment (shanamatthews) [inline src/platforms/javascript/common/session-replay/configuration.mdx:28]: > before the SDK should start flushing to Sentry.

Could we simply say "...start sending to Sentry"? "flushing to Sentry" sounds a bit weird at first :)

**Review by shanamatthews — APPROVED** [review]

File: src/platforms/javascript/common/session-replay/configuration.mdx (hunk 1 above)
Comment (mydea) [reply src/platforms/javascript/common/session-replay/configuration.mdx:28]: technically the default is 4999 (one ms shorter than the default flush duration)!

Comment (bruno-garcia) [inline src/platforms/javascript/common/session-replay/configuration.mdx:28]:
```suggestion
| minReplayDuration       | number                   | 5000     | The length of the replay, **in milliseconds**, before the SDK should start sending to Sentry. Max value is 15000.                                            |
```

Comment (bruno-garcia) [reply src/platforms/javascript/common/session-replay/configuration.mdx]: @mydea will leave as 5 for simplicity sake

**Pull request merged**